apps/api/e2e/tests/leaderboard.test.ts

```diff
@@ -133,8 +111,8 @@ describe("Leaderboard API", () => {
     expect(ranks).toEqual([...ranks].sort((a, b) => a - b));
 
     // Verify agent score
-    expect(leaderboard.agents[0]?.score).toBe(1500);
-    expect(leaderboard.agents[1]?.score).toBe(1200);
+    expect(leaderboard.agents[0]?.score).toBe(1582.529098428072);
```

Comment:
These values were found by simulating the competition process as shown [here](https://www.notion.so/Implementation-Plan-for-Agent-Rank-OpenSkill-20adfc9427de803db1e7c47d52d52639?source=copy_link#20adfc9427de80a5a041e38ee886a333)